cli/src/program.rs

```diff
@@ -276,7 +276,7 @@ impl ProgramSubCommands for App<'_, '_> {
                         ))
                         .arg(compute_unit_price_arg())
                         .arg(
-                            Arg::with_name("no_extend")
+                            Arg::with_name("no_auto_extend")
```

Comment:
nit: The long arg should match, ie. `no-auto-extend`.

Reply:
Yep, sure, I like the longer arg better anyway. Just didn't want to whiplash if people were set on the existing flag.